test/net/sourceforge/kolmafia/KoLAdventureValidationTest.java

```diff
@@ -1368,7 +1356,33 @@ void cannotAdventureWhenQueenHeartIsHandedIn() {
               withProperty("sidequestOrchardCompleted", "frat"));
```

Comment:
Technically, IslandManager sets this to "hippy" or "fratboy", not "frat".
In retrospect, "frat" would have been better.

But since you correctly check for "sidequestOrchardCompleted" != "none", it doesn't really matter. :)

Reply:
Fixed anyway